packages/sql-pg/src/PgClient.ts

```diff
@@ -80,11 +101,15 @@ export interface PgClientConfig {
   readonly prepare?: boolean | undefined
   readonly types?: Record<string, postgres.PostgresType> | undefined
 
-  readonly debug?: postgres.Options<{}>["debug"] | undefined
+  readonly debug?: PostgresOptions["debug"] | undefined
 }
 
 type PartialWithUndefined<T> = { [K in keyof T]?: T[K] | undefined }
 
+interface PostgresOptions extends postgres.Options<{}> {
```

Comment:
This should be exported if we are using it on a public interface. Or you can revert the "debug" change.